.gitignore

```diff
@@ -9,3 +9,4 @@ creds.json
 uv.lock
 .venv
 .ruff_cache
+tokens.json
```

Comment:
`tokens.json` could probably be automatically generated by a script that's hooked in `Makefile` e.g. `make generate-tokens`

Reply:
I will figure out the best way to make tokens on scale